apps/mull-api/src/app/event/event.service.ts

```diff
@@ -18,6 +20,58 @@ export class EventService {
     return this.eventRepository.findOne(id);
   }
 
+  findHostEvents(userId: number): Promise<Event[]> {
+    return this.eventRepository.find({
+      relations: ['host'],
+      where: {
+        host: {
+          id: userId,
+        },
+      },
+    });
+  }
+
+  findCoHostEvents(userId: number): Promise<Event[]> {
+    return this.eventRepository
+      .createQueryBuilder('event')
+      .leftJoin('event.coHosts', 'user')
+      .where('user.id = :userId', { userId })
+      .getMany();
+  }
+
+  findJoinedEvents(userId: number): Promise<Event[]> {
+    return this.eventRepository
+      .createQueryBuilder('event')
+      .leftJoin('event.participants', 'user')
+      .where('user.id = :userId', { userId })
+      .getMany();
+  }
+
+  findDiscoverEvent(userId: number): Promise<Event[]> {
```

Comment:
This name doesn't make it easy for me to know what this method does. Could you add some documentation for this method, please?

Reply:
Of course I can. It finds all the events the user has joined

Reply:
Doesn't it find the events that the user has not joined, and is not the host/cohost of?

Reply:
Oh sorry, I thought I saw findJoinedEvent. Yes you're right